tests/test_platform.py

```diff
@@ -74,7 +74,6 @@ def test_dump_runcard(platform):
         "iq_angle",
         "mean_gnd_states",
         "mean_exc_states",
-        "classifiers_hpars",
```

Comment:
Thanks for opening this. I believe this `@pytest.mark.parametrize` can be removed completely, it is leftover from the `test_update` that was removed.